src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,638 @@
+   */
+  tickLabels?: (string | JSX.Element)[];
+  /** Optional label for the minimum value. If undefined then the min railDomain value is shown. Use empty string for no label.
+   * @default undefined
+   */
+  minLabel?: string | JSX.Element;
+  /** Optional label for the maximum value. If undefined then the max railDomain value is shown. Use empty string for no label.
+   * @default undefined
+   */
+  maxLabel?: string | JSX.Element;
+  /** Defines the allowed behavior when moving Thumbs when multiple Thumbs are shown.  Possible values:
+   * 'allow-crossing' - allows thumb to cross other thumbs. Default.
+   * 'step-separated'- keeps the thumb from crossing and separated by a step.
+   * @default 'allow-crossing'
+   */
+  thumbMode?: ThumbMoveMode;
+  /** optional props to include on slider container */
+  containerProps?: StylingProps;
+  /**
+   * Callback fired when the value(s) of the slider has changed. This will receive changes at
+   * the end of a slide as well as changes from clicks on rails and tracks.
+   */
+  onChange?: (values: ReadonlyArray<number>) => void;
+  /** Callback fired when the value(s) of the slider are internally updated during operations
+   * like dragging a Thumb. Use this callback with caution as a high-volume updates will occur
+   * when dragging.
+   */
+  onUpdate?: (values: ReadonlyArray<number>) => void;
+};
+
+/**
+ * Slider component that display Thumbs for each value specified along a Rail.
+ * @example
+ * <Slider values={[10]} railDomain={{ min: 0, max: 60 }} disabled />
+ * <Slider values={[10, 20]} railDomain={{ min: 0, max: 50 }} step={2} />
+ * <Slider values={[10, 20, 30, 40]} railDomain={{ min: 0, max: 60 }} setFocus thumbMode='step-separated' />
+ */
+export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
+  (props, ref) => {
+    const {
+      railDomain,
+      values,
+      step = 1,
+      setFocus = false,
+      toolTipFunction,
+      disabled = false,
+      tickLabels,
+      minLabel,
+      maxLabel,
+      trackDisplayMode = undefined,
+      thumbMode = 'allow-crossing',
+      toolTipPosition = 'top',
+      containerProps = {},
+      onChange,
+      onUpdate,
+      hideTooltip = false,
+    } = props;
+
+    const {
+      className: containerClassName,
+      ...remainingContainerProps
+    } = containerProps;
+
+    const [currentValues, setCurrentValues] = React.useState(values);
+    React.useEffect(() => {
+      setCurrentValues(values);
+    }, [values]);
+
+    const [minValueLabel, setMinValueLabel] = React.useState(minLabel);
+    React.useEffect(() => {
+      setMinValueLabel(minLabel ?? railDomain.min.toString());
+    }, [minLabel, railDomain]);
+
+    const [maxValueLabel, setMaxValueLabel] = React.useState(maxLabel);
+    React.useEffect(() => {
+      setMaxValueLabel(maxLabel ?? railDomain.max.toString());
+    }, [maxLabel, railDomain]);
+
+    const [trackDisplay, settTackDisplay] = React.useState<TrackDisplayMode>(
```

Comment:
mistype

Reply:
fat fingered 't' removed.

Reply:
There's another typo 😄 it should be `setTrackDisplay` (Track not Tack)

Reply:
Oh Lord!... hopefully third time is a charm.

Reply:
@bsteinbk why dont you try to use this one? :D 
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
'Tack' is a valid English work .... but it would have flagged 'sett'

Reply:
well..hey, cant say I havent tried to help you :D

Reply:
Cause "work" is also valid, even though you meant "word"..
Need some AI here...

Reply:
I just need the "I" part of "AI"